es-bs-base/scss/_variables.scss

```diff
@@ -1511,26 +1511,26 @@ $form-feedback-tooltip-border-radius: $tooltip-border-radius !default;
 // Popovers
 
 $popover-font-size:                 $font-size-sm !default;
-$popover-bg:                        $gray-900 !default;
+$popover-bg:                        $black !default;
```

Comment:
This is a pre-existing issue, but I noticed that if you use one of the [complicated examples from BootstrapVue](https://bootstrap-vue.org/docs/components/popover#advanced-b-popover-usage-with-reactive-content) for the default `b-popover`, the styling doesn't look right as a result of the dark theme being default. @ericdouglaspratt do you think it would make sense to change the default in the SCSS to the light theme?

<img width="764" alt="Screenshot 2024-05-06 at 12 06 57" src="https://github.com/EnergySage/es-ds/assets/4626345/d1d6b28e-afd8-4785-bf3e-fd864d29e9d0">

Reply:
I sincerely hope we're not putting forms in popovers anywhere, because popovers are meant to be extremely temporary and can too easily be accidentally dismissed and you lose data that you've entered.

As the default page background is white, it makes sense to me that most cases of the popover should be black, and therefore the default should be black.